### DISCUSSION TITLE
Inquiry About Security Onion Compatibility with Malwarebytes and Antivirus Protocols

### DISCUSSION THREAD

**Discussion by aliakanan12** [Security-Onion-Solutions/securityonion#13446] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

20

### RAM

256

### Storage for /

5tb

### Storage for /nsm

2tb

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

I am writing to seek clarification regarding the compatibility of Security Onion with other security applications, specifically Malwarebytes. Additionally, I am interested in understanding how Security Onion might interact with antivirus protocols in general.

Could you please provide information on the following points?

Are there any known conflicts or issues when running Security Onion alongside Malwarebytes?
Does the presence of Security Onion affect the functionality or efficiency of standard antivirus protocols?
Are there recommended configurations or best practices to ensure optimal performance when using Security Onion with other security applications?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by reyesj2** [comment]
We recommend avoiding installing third party agents 
https://docs.securityonion.net/en/2.4/best-practices.html#avoid-third-party-software-and-modifications
